EventFilter/EcalRawToDigi/plugins/DeclsForKernels.h

```diff
@@ -40,11 +41,10 @@ namespace ecal {
       // FIXME: we should separate max channels parameter for eb and ee
       // FIXME: replace hardcoded values
       void allocate(ConfigurationParameters const &config, cudaStream_t cudaStream) {
-        digisEB.data = cms::cuda::make_device_unique<uint16_t[]>(config.maxChannels, cudaStream);
-        digisEE.data = cms::cuda::make_device_unique<uint16_t[]>(config.maxChannels, cudaStream);
```

Comment:
Maybe the missing *10 here is also why in the past we were not able to validate fully the MC workflow?
... btw, not it should be fixed.